cmd/ci-secret-generator/main.go

```diff
@@ -195,6 +224,12 @@ func updateSecrets(config secretgenerator.Config, client secrets.Client) error {
 				errs = append(errs, errors.New(msg))
 				continue
 			}
+			if err := validateCommandOutput(out, field.Validation); err != nil {
+				msg := fmt.Sprintf("failed to validate command output:\n%s", string(out))
```

Comment:
We definitely _do not_ want to print `out` here.

Reply:
If secret leaking is the concert you're definitely right. Shouldn't the same  be applied [here](https://github.com/openshift/ci-tools/blob/master/cmd/ci-secret-generator/main.go#L194) by the way? `err` could (even though I'm not 100% sure) leak something and no censoring has been applied as well.